app/celer-sim/Transporter.cc

```diff
@@ -34,6 +34,22 @@ namespace celeritas
 {
 namespace app
 {
+namespace
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Print progress after N events when requested.
+ */
+void progress(EventId const id, size_type const num_primaries)
```

Comment:
```suggestion
void log_progress(EventId const id, size_type const num_primaries)
```

Otherwise one might think `progress` is a verb (pro-GRESS) rather than noun (PRO-gress) .. isn't English fun?!